bpf/tests/pktgen.h

```diff
@@ -67,6 +67,13 @@ volatile const __u8 mac_zero[] = mac_zero_addr;
 #define v4_pod_two	IPV4(192, 168, 0, 2)
 #define v4_pod_three	IPV4(192, 168, 0, 3)
 
+#define v4_pod_one_on_node_two		IPV4(192, 168, 1, 1)
```

Comment:
After https://github.com/cilium/cilium/pull/40294, we haven't come up yet with a good practise to keep this file in sync with the newly introduced https://github.com/cilium/cilium/blob/main/bpf/tests/scapy/pkt_defs.py.
Maybe it is worth adding these bits also there?

Reply:
oh snap ... yes, we'll need to see about porting the existing tests over then I guess. Will add it for now.